galsim/wfirst/wfirst_detectors.py

```diff
@@ -122,6 +154,14 @@ def allDetectorEffects(img, rng=None, exptime=None):
     dark_noise = galsim.DeviateNoise(galsim.PoissonDeviate(rng, dark_current))
     img.addNoise(dark_noise)
 
+    # Persistence (use WFIRST coefficients)
+    applyPersistence(img,prev_exposures)
+
+    # Update the 'prev_exposures' queue
+    if len(prev_exposures)>len(galsim.wfirst.persistence_coefficients):
```

Comment:
I'm pretty sure this is wrong.  It should be `>=`.  Or you can leave it like this, but move the insert command to happen before this check.